README.md

```diff
@@ -153,6 +161,14 @@ export SQLA_CONN=<psql:address-to-box>
    
    *Note: 18F team members should not set these environment variables. 18F and FEC team members will also have additional environment variables to set up. Please reach out to a team member for detailed information.*
 
+4. If you are using database replicas/mirrors you can also restrict connections to them to be asynchronous tasks only by running:
+
+   ```
+   export SQLA_RESTRICT_FOLLOWER_TRAFFIC_TO_TASKS=True
```

Comment:
Using `True` here implies that the value of the environment variable might be evaluated literally, so that valid options would be `True` or `False`. But this is actually a string passed to `bool`, so any non-empty string will become `True`, including `False`. Might want to either change the parsing logic or change the example value for clarity.